### DISCUSSION TITLE
Failed to install Win10/11 into iSCSI with NUC9 UEFI

### DISCUSSION THREAD

**Discussion by imyzcn** [ipxe/ipxe#602] [Q&A]
Hi, folks,

After a bug existing in VMware UEFI(https://github.com/ipxe/ipxe/discussions/600) had been told, I was trying to install Win10/11 into iSCSI target with my Intel NUC9 these days. Hardwares & Softwares are as follows:

Intel NUC9 (Dual NICs: I219-LM-7 & I210)
Tiny PXE Server 1.0.0.23
ipxe-x86_64.efi: iPXE 1.21.1+ (g16d95)
Wimboot 2.7.4

First of all, I downloaded the latest NIC drivers from Intel, and then had them injected into the boot.wim(index:1 & index:2) copied from Windows ISO image by the following commands:

dism /mount-wim /wimfile:XX /index:1(2) /mountdir:YY
dism /image:XX /add-driver /driver:ZZ.inf
dism /unmount-wim /mountdir:YY /commit

And drivers injected:
e1d65x64.sys / e1d68x64.sys (i219-lm-7)
e1r65x64.sys / e1r68x64.sys (i210)

After that, I put the boot.wim into the correct folder of Tiny PXE Server, and then power-on my NUC9 and boot from NIC. The sanhook directive worked fine, and the iSCSI target was recognized as a local disk properly after I chain-loaded the above-mentioned boot.wim by Wimboot with following script:

sanhook ${root-path}
kernel wimboot
initrd boot/bcd         BCD
initrd boot/boot.sdi    boot.sdi
initrd sources/boot.wim boot.wim
boot

But, the things next are not that easy as I thought. I can’t get rid of this ‘Famous’ BSOD, and always end-up with the error INACCESSIBLE_BOOT_DEVICE…… NDIS.SYS…

I’ll be grateful if anybody could give me a guiding hand and tell me which step I took was wrong?

Best regards

**Comment by NiKiZe** [comment]
There is several discussions on the topic already. I would think that a search for iscsi here on the iPXE GitHub project would find them for you?
